pkg/civo/objectStorage.go

```diff
@@ -71,7 +71,7 @@ func (c *CivoConfiguration) GetAccessCredentials(credentialName string, region s
 			return civogo.ObjectStoreCredential{}, err
 		}
 
-		for i := 0; i < 30; i++ {
+		for i := 0; i < 12; i++ {
```

Comment:
I was thinking 2 minutes was plenty of timeout. If we leave it at 30 then that is 5 minutes which seems a bit extreme since most of the time the credentials are found after just one loop.